server/src/main/java/org/elasticsearch/action/fieldcaps/TransportFieldCapabilitiesAction.java

```diff
@@ -371,10 +371,13 @@ private FieldCapabilitiesIndexResponse shardOperation(final FieldCapabilitiesInd
                         if (searchExecutionContext.getFieldType(parentField) == null) {
                             // no field type, it must be an object field
                             ObjectMapper mapper = searchExecutionContext.getObjectMapper(parentField);
```

Comment:
Does it makes sense to add `@Nullable` to the `searchExecutionContext.getObjectMapper` method so you get a FindBugs warning if you don't check the result for `null`?